plugins/osx/osx.plugin.zsh

```diff
@@ -161,6 +161,15 @@ function itunes() {
 		vol)
 			opt="set sound volume to $1" #$1 Due to the shift
 			;;
+		playing)
+			local state=`osascript -e 'tell application "iTunes" to player state as string'`
+			if [ "$state" = "playing" ]; then
+				currenttrack=`osascript -e 'tell application "iTunes" to name of current track as string'`
+				currentartist=`osascript -e 'tell application "iTunes" to artist of current track as string'`
+				echo "You're currently litening to $fg[yellow]$currenttrack$reset_color by $fg[yellow]$currentartist$reset_color";
```

Comment:
There's a spelling error: litening → listening

I would also use [Prompt expansion colors](http://zsh.sourceforge.net/Doc/Release/Prompt-Expansion.html#Visual-effects) instead of `$fg`/`$reset_color`, as well as drop the `You're currently...` part, like so:
`print -P "Listening to '%F{yellow}Melancholy Hill%f' by %F{blue}Gorillaz%f"`

I cannot try if the osascript works because I'm on Linux, but @apjanke can confirm.
:+1: